### ISSUE TITLE
adjust pagination tap target size

### ISSUE THREAD

**Issue by timelsass** [BoldGrid/boldgrid-theme-framework#105]
Mentioned as part of user feedback, the tap target size for tablets and mobile should be increased for pagination.  The bulk of the accessibility work will probably take place in #63, but wanted to be sure the user knew this will be updated.

For reference I think this repo should follow the [fundamentals guide for accessible styles provided by Google](https://developers.google.com/web/fundamentals/accessibility/accessible-styles).

The tap target size should give a surface area around 9mm to avoid finger padding collisions, Ideally something 48px x 48px  or larger.  I think an increase in the link/span padding for the pagination should work.
